deploy/gke-marketplace-app/server-deployer/build_and_push.sh

```diff
@@ -28,7 +28,7 @@ export REGISTRY=gcr.io/$(gcloud config get-value project | tr ':' '/')
 export APP_NAME=tritonserver
 export MAJOR_VERSION=2.8
 export MINOR_VERSION=2.8.6
-export NGC_VERSION=21.03-py3
+export NGC_VERSION=21.04-py3
```

Comment:
Don't update anything in gke-marketplace-app for now. @mengdong should we make these kind of updates for each release?

Reply:
I can make an update for 21.04 tomorrow. There are a few other places that need to be updated.

Reply:
and yes, please keep this one.